src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/PaymentOrder/Abort/AbortTests.cs

```diff
@@ -26,7 +26,7 @@ public async Task Abort_PaymentOrder(Product[] products)
 
             var orderLink = message.OriginalString.Substring(message.OriginalString.IndexOf("/")).Replace(" has been Aborted", "");
 
-            var order = await SwedbankPayClient.PaymentOrder.Get(new Uri(orderLink, UriKind.RelativeOrAbsolute), SwedbankPay.Sdk.PaymentOrders.PaymentOrderExpand.All);
+            var order = await SwedbankPayClient.Get(new Uri(orderLink, UriKind.RelativeOrAbsolute), SwedbankPay.Sdk.PaymentOrders.PaymentOrderExpand.All);
```

Comment:
Is it possible to have this read like the following?

```suggestion
            var order = await SwedbankPayClient.PaymentOrders.Get(new Uri(orderLink, UriKind.RelativeOrAbsolute), SwedbankPay.Sdk.PaymentOrders.PaymentOrderExpand.All);
```

Reply:
I've ripped out that part of the `SwedbankPayClient`. If you want to use one singular part of the api at a time you could just use `IPaymentOrdersClient` instead of the `ISwedbankPayClient`.